metabase/metabase
# Add models educational modal to Admin > Data Model

Adds an educational modal telling more about Models to Admin > Data Model page.
~⚠️ Still need to add a URL to the Learn article~ ([done](https://github.com/metabase/metabase/pull/19832/commits/736633d443757971af49042fa1880326444ad400))

### To Verify

1. Sign in as admin
2. Open `/admin/datamodel`
3. Click "Simplify your schema with Models" at the top right
4. You should see an educational modal and should be able to close it

~Note: "Learn how" button is supposed to be a link to a Learn article which is still missing~ ([done](https://github.com/metabase/metabase/pull/19832/commits/736633d443757971af49042fa1880326444ad400))

### Screenshot

<img width="1333" alt="CleanShot 2022-01-21 at 14 36 00@2x" src="https://user-images.githubusercontent.com/17258145/150528666-dff21034-40a6-4c67-8092-ba824036673f.png">
